sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1028,6 +1033,18 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
        spval,recname(13),qqs(ista_2l,jsta_2l,1),lm)
        call read_netcdf_3d_para(ncid3d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
        spval,recname(14),qqg(ista_2l,jsta_2l,1),lm)
+       call read_netcdf_3d_para(ncid3d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+       spval,recname(15),smoke(ista_2l,jsta_2l,1,1),lm)
+       call read_netcdf_3d_para(ncid3d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+       spval,recname(16),fv3dust(ista_2l,jsta_2l,1,1),lm)
+       call read_netcdf_3d_para(ncid2d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+       spval,recname(17),extsmoke(ista_2l,jsta_2l,1),lm)
+       call read_netcdf_3d_para(ncid2d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+       spval,recname(18),extdust(ista_2l,jsta_2l,1),lm)
+       call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u,       &
+       spval,recname(19),ebb(ista_2l,jsta_2l))
```

Comment:
@EricJames-NOAA Could you move reading ebb and hwp after line 1610? They should be in the block of reading variables from FV3 phy files. Please use the consistent format e.g. 
      VarName='maxvort02'
      call read_netcdf_2d_para(ncid3d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
      spval,VarName,rel_vort_max(ista_2l,jsta_2l))

Reply:
@WenMeng-NOAA thanks for the suggestion, I made this change